### PR TITLE
getEncryptionPublicKey

### PR DESCRIPTION
**Pull request by logvik**
We need it for https://github.com/MetaMask/metamask-extension/issues/1190

### CODE REVIEW

**Comment by logvik** [comment]
@danfinlay ,  do you know why ci/circleci: test takes so long time? And how can we boost this process?

**Comment by danfinlay** [comment]
CircleCI has been really slow this last week, I believe there’s an issue on their end, there is a long queue before running the tests. I hope they sort it out soon.

File: test/index.js [hunk 1]
```diff
@@ -335,22 +335,32 @@ describe('simple-keyring', () => {
       assert.equal(restored, address, 'recovered address')
     })
   })
-
+
```
Comment (lazaridiscom) [inline test/index.js]: space accident

**Comment by lazaridiscom** [comment]
You need to rebase/squash your commits into one (personally, I sometimes mess it up, close the PR and create a new one with a clean commit).

**Comment by logvik** [comment]
@lazaridiscom I did squash

**Comment by lazaridiscom** [comment]
follow-up #32

**Pull request closed without merging**